utils/retry_policies.ts

```diff
@@ -0,0 +1,17 @@
+import { RetryOptions } from "durable-functions";
+
+export const internalRetryOptions: RetryOptions = {
+  backoffCoefficient: 1.5,
+  firstRetryIntervalInMilliseconds: 500,
+  maxNumberOfAttempts: 10,
+  maxRetryIntervalInMilliseconds: 3600 * 100,
```

Comment:
what's the benefit of specifying `maxRetryIntervalInMilliseconds` and `retryTimeoutInMilliseconds`?

Reply:
They are mandatory. Another way is to use the constructor then modify the returned object (I'm gonna refactor this)